Gemfile

```diff
@@ -8,3 +8,4 @@ gem "cucumber"
 gem "rake"
 gem "rspec-rails"
 gem "standard"
+gem "factory_bot", github: "thoughtbot/factory_bot", branch: "compile_factory_instrumentation"
```

Comment:
Remove this line when https://github.com/thoughtbot/factory_bot/pull/1587 is merged and available as part of a release.

Reply:
> I assume this is because factory_bot isn't released yet.

@mike-burns I was hoping that this line would include the changes necessary to make CI pass. Do you know of a separate mechanism that's still loading the published gem?

Reply:
Oh you're right, I overlooked that you're hardcoding the branch name in here.

I no longer have a guess as to why CI is failing!